frappe/tests/test_data_import.py

```diff
@@ -1,89 +1,89 @@
-# Copyright (c) 2015, Frappe Technologies Pvt. Ltd. and Contributors
```

Comment:
delete this file instead of commenting it. Make sure all test cases are replicated in the new tests